api/wasm/indigo-ketcher/indigo-ketcher.cpp

```diff
@@ -503,44 +501,50 @@ namespace indigo
             if (selected_atoms.size())
                 base += indigoCountAtoms(id);
         }
-
-        molecularWeightStream << VectorStringBracketed(molWeights);
-        mostAbundantMassStream << VectorStringBracketed(mamMasses);
-        monoisotopicMassStream << VectorStringBracketed(misoMasses);
-        massCompositionStream << VectorStringBracketed(massCompositions);
-        grossFormulaStream << VectorStringBracketed(grossFormulas);
     }
 
     void calculate_reaction(const IndigoKetcherObject& iko, std::stringstream& molecularWeightStream, std::stringstream& mostAbundantMassStream,
                             std::stringstream& monoisotopicMassStream, std::stringstream& massCompositionStream, std::stringstream& grossFormulaStream,
                             const std::vector<int>& selected_atoms)
     {
+        std::vector<std::string> molWeights[2], mamMasses[2], misoMasses[2], massCompositions[2], grossFormulas[2];
```

Comment:
To make design clear, let's introduce, for named idexes for products and reactants, example:
```
enum { IDX_PRODUCTS = 0, IDX_REACTANTS = 1 };
```
and use them instead [0] and [1]. 
Then it will give a clue why each vector is 2-ice and what are their roles.

Reply:
fixed